src/01.UI-Elements/UndoButton.md

```diff
@@ -0,0 +1,3 @@
+```js padded
+<UndoButton visibility="shown"/>
+```
```

Comment:
I cant find the visibility anywhere else? What is it for?

Reply:
Good point. Thanks! I forgot to change it here.